assets/thankyou.css

```diff
@@ -19,136 +24,149 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  gap: 16px;
+  margin-bottom: 33px;
+}
+.thankyou-page-container .thankyou-wrapper .content-wrapper .image {
+  width: 54px;
+  height: 54px;
+  object-fit: contain;
 }
 .thankyou-page-container .thankyou-wrapper .content-wrapper .heading {
-  line-height: 42px;
+  text-align: center;
 }
 .thankyou-page-container .thankyou-wrapper .content-wrapper .heading .title {
   display: block;
-  font-size: 24px;
+  font-size: 30px;
+  font-weight: 800;
+  line-height: 140%;
 }
-@media (min-width: 768px) {
-  .thankyou-page-container .thankyou-wrapper .content-wrapper .heading .title {
-    font-size: 30px;
-  }
-}
-.thankyou-page-container .thankyou-wrapper .content-wrapper .icon, .thankyou-page-container .thankyou-wrapper .content-wrapper img {
-  width: 66px;
-  color: #8DC447;
-  height: 66px;
-  margin-bottom: 16px;
+.thankyou-page-container .thankyou-wrapper .content-wrapper .heading .subtitle {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 140%;
 }
 .thankyou-page-container .thankyou-wrapper .content-wrapper .description {
-  margin-top: 15px;
-  opacity: 0.9;
+  font-size: 14px;
   text-align: center;
-  font-size: 15px;
   font-weight: 400;
-  line-height: 1.5rem;
+  line-height: normal;
 }
-@media (min-width: 768px) {
-  .thankyou-page-container .thankyou-wrapper .content-wrapper .description {
-    font-size: 16px;
-    line-height: 2rem;
-  }
+.thankyou-page-container .cart-result-section .cart-items {
+  margin-bottom: 6px;
 }
-.thankyou-page-container .thankyou-wrapper .content-wrapper .yc-btn {
-  margin-top: 2rem;
+.thankyou-page-container .cart-result-section .cart-items .product-list {
+  display: grid;
+  gap: 6px;
 }
-.thankyou-page-container .order-details {
-  border: 1px solid #EBEBEB;
-  padding: 24px 28px;
-  border-radius: 0 0 4px 4px;
+.thankyou-page-container .cart-result-section .cart-items .product-list .product-item {
+  display: flex;
+  gap: 16px;
+  background: white;
+  padding: 20px 24px;
+}
+.thankyou-page-container .cart-result-section .cart-items .product-list .product-item .product-image {
+  width: 62px;
+  height: 62px;
+  object-fit: contain;
```

Comment:
add object-fit: cover  so that we can make sure the image will fill the height and width of the box  and it will maintaining its aspect ratio

Reply:
**object-fit : cover** 
<img width="462" alt="Screen Shot 2023-10-09 at 14 23 14" src="https://github.com/youcan-shop/cod-theme-3/assets/93322743/29d3939c-a793-4458-8a02-e47ba8126e03">

**object-fit: contain**
<img width="462" alt="Screen Shot 2023-10-09 at 14 24 40" src="https://github.com/youcan-shop/cod-theme-3/assets/93322743/26098216-661c-434d-9c9e-e860a6c6e5ce">

as you see in the two images the result is clear since the box is too small 62x62 and doesn't have a border to display the limit of the box we don't need to use it

Reply:
contain
<img width="430" alt="Screenshot 2023-10-09 at 14 39 48" src="https://github.com/youcan-shop/cod-theme-3/assets/108662497/4c622a9d-c717-48e8-b2bb-bf5a1a4d1c8c">
cover
<img width="430" alt="Screenshot 2023-10-09 at 14 39 59" src="https://github.com/youcan-shop/cod-theme-3/assets/108662497/b4fa416a-4bbf-4695-b19c-46db85d9e7d6">

Reply:
yes while most of the images doesn't have same width and height this is why we need to add `object-fit: cover` so that any image can scale within its container

Reply:
Either way the seller has to make sure the image looks good. With that said, this LGTM